runtime/src/bank.rs

```diff
@@ -1054,6 +1054,12 @@ struct VoteWithStakeDelegations {
     delegations: Vec<(Pubkey, (StakeState, AccountSharedData))>,
 }
 
+struct LoadVoteAndStakeAccountsResult {
+    vote_with_stake_delegations_map: DashMap<Pubkey, VoteWithStakeDelegations>,
+    invalid_stake_keys_set: DashSet<Pubkey>,
+    invalid_vote_keys_set: DashSet<Pubkey>,
```

Comment:
If we're looking to prevent the cache entering an inconsistent state, it might be valuable to tag the addresses with the reason they are invalid; missing, wrong owner, bad state

Reply:
Add reasons, thanks for the suggestion